data_structures/route_parameters.cpp

```diff
@@ -36,7 +36,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 RouteParameters::RouteParameters()
     : zoom_level(18), print_instructions(false), alternate_route(true), geometry(true),
-      compression(true), deprecatedAPI(false), uturn_default(false), classify(false),
+      compression(true), deprecatedAPI(false), uturn_default(false), classify(false), mapped_points(true),
```

Comment:
Please remove the `mapped_points` parameter entirely. It has no significant implications on size or performance. I'm only talking about the parameter for the API not the feature as such - it should just be the normal behavior and not user configurable.